### PR TITLE
Add a workaround for core table altering

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add a workaround when a contributor wants to alter the core table
| Fixed ticket? | /

### CODE REVIEW

File: src/content/1.7/modules/sell/techvalidation-checklist.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ Additionally, distribution licenses like CC-0 or CC-by-sa are appropriate for ar
 
 Module may create all the table they need in the database. However altering core table is forbidden.
 
+In case you want to add columns for a existing object, the workaround is to create a new table with a foreign key targetting the primary key of the core table.
```
Comment (kpodemski) [inline src/content/1.7/modules/sell/techvalidation-checklist.md:37]: https://www.a-or-an.com/a_an/existing :)

Isn't it a "new field", not "new column" for Object?

Comment (Quetzacoalt91) [reply src/content/1.7/modules/sell/techvalidation-checklist.md]: Thanks for the feedback, I fixed the typo.

In this case we are talking about the database and not ObjectModels, that's why I was talking about `columns`

Comment (kpodemski) [reply src/content/1.7/modules/sell/techvalidation-checklist.md]: Edited text looks better 👍

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @Quetzacoalt91

**Pull request merged**